tests/test_arthur.py

```diff
@@ -61,7 +62,54 @@ def setUp(self):
     def tearDown(self):
         self.conn.flushdb()
 
+    def test_add_task(self):
+        """Check whether tasks are added"""
+
+        app = Arthur(self.conn, async_mode=False)
+
+        initial_tasks = len(app._tasks.tasks)
+        app.add_task("arthur.task", "backend", {"a": "a", "b": "b"})
```

Comment:
I would check that the task was properly created instead of checking the length of the tasks list.